users/serializers.py

```diff
@@ -1,35 +1,48 @@
-import datetime
-from itertools import islice
-
 from django.contrib.auth import get_user_model, authenticate
 from rest_framework import serializers
 from rest_framework.authtoken.models import Token
 from users.models import UserTag
 
-class UserRegistrationSerializers(serializers.ModelSerializer):
-    tag = serializers.CharField(write_only=True)
+from .utils import generateHandle
```

Comment:
there is no camelCase in python.

functions and variables = snake_case
classes = PascalCase